doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1668,51 +1668,6 @@ name of the variable (here :g:`n`) is chosen based on :g:`T`.
    then required to prove that False is indeed provable in the current
    context. This tactic is a macro for :n:`elimtype False`.
 
-Checking properties of terms
-----------------------------
-
-Each of the following tactics acts as the identity if the check
-succeeds, and results in an error otherwise.
-
-.. tacn:: constr_eq @term @term
-   :name: constr_eq
-
-   This tactic checks whether its arguments are equal modulo alpha
-   conversion, casts and universe constraints. It may unify universes.
-
-.. exn:: Not equal.
-   :undocumented:
-
-.. exn:: Not equal (due to universes).
-   :undocumented:
-
-.. tacn:: constr_eq_strict @term @term
-   :name: constr_eq_strict
-
-   This tactic checks whether its arguments are equal modulo alpha
-   conversion, casts and universe constraints. It does not add new
-   constraints.
-
-.. exn:: Not equal.
-   :undocumented:
-
-.. exn:: Not equal (due to universes).
-   :undocumented:
-
-.. tacn:: unify @term @term
-   :name: unify
-
-   This tactic checks whether its arguments are unifiable, potentially
-   instantiating existential variables.
-
-.. exn:: Unable to unify @term with @term.
-   :undocumented:
-
-.. tacv:: unify @term @term with @ident
-
-   Unification takes the transparency information defined in the hint database
-   :n:`@ident` into account (see :ref:`the hints databases for auto and eauto <thehintsdatabasesforautoandeauto>`).
-
 .. tacn:: is_evar @term
```

Comment:
The move of the section is incomplete, the section continues from `is_evar` until `is_var`. In #15015, I've noticed that you've also appended some related tactics after that (`is_cofix`, etc.). These should also be added here, rather than there.